azure-kusto-ingest/tests/e2e.py

```diff
@@ -36,60 +35,65 @@ def __init__(self):
     def create_deft_table_csv_mappings():
         """A method to define csv mappings to deft table."""
         mappings = list()
-        mappings.append(CsvColumnMapping(columnName="rownumber", cslDataType="int", ordinal=0))
-        mappings.append(CsvColumnMapping(columnName="rowguid", cslDataType="string", ordinal=1))
-        mappings.append(CsvColumnMapping(columnName="xdouble", cslDataType="real", ordinal=2))
-        mappings.append(CsvColumnMapping(columnName="xfloat", cslDataType="real", ordinal=3))
-        mappings.append(CsvColumnMapping(columnName="xbool", cslDataType="bool", ordinal=4))
-        mappings.append(CsvColumnMapping(columnName="xint16", cslDataType="int", ordinal=5))
-        mappings.append(CsvColumnMapping(columnName="xint32", cslDataType="int", ordinal=6))
-        mappings.append(CsvColumnMapping(columnName="xint64", cslDataType="long", ordinal=7))
-        mappings.append(CsvColumnMapping(columnName="xuint8", cslDataType="long", ordinal=8))
-        mappings.append(CsvColumnMapping(columnName="xuint16", cslDataType="long", ordinal=9))
-        mappings.append(CsvColumnMapping(columnName="xuint32", cslDataType="long", ordinal=10))
-        mappings.append(CsvColumnMapping(columnName="xuint64", cslDataType="long", ordinal=11))
-        mappings.append(CsvColumnMapping(columnName="xdate", cslDataType="datetime", ordinal=12))
-        mappings.append(CsvColumnMapping(columnName="xsmalltext", cslDataType="string", ordinal=13))
-        mappings.append(CsvColumnMapping(columnName="xtext", cslDataType="string", ordinal=14))
-        mappings.append(CsvColumnMapping(columnName="xnumberAsText", cslDataType="string", ordinal=15))
-        mappings.append(CsvColumnMapping(columnName="xtime", cslDataType="timespan", ordinal=16))
-        mappings.append(CsvColumnMapping(columnName="xtextWithNulls", cslDataType="string", ordinal=17))
-        mappings.append(CsvColumnMapping(columnName="xdynamicWithNulls", cslDataType="dynamic", ordinal=18))
+        mappings.append(ColumnMapping(columnName="rownumber", cslDataType="int", properties={MappingConsts.ORDINAL.value:"0"}))
```

Comment:
What is the recommended way to create mapping? on one hand we give this example and on the other hand you've added functions to edit properties

Reply:
e2e is not a sample - but anyway i think both ways are ok, u have to know things to come up with this way of setting the properties